.github/workflows/codeql-analysis.yml

```
@@ -11,59 +10,58 @@ on:
   push:
     # Dependabot triggered push events have read-only access, but uploading code
     # scanning requires write access.
-    branches-ignore: [dependabot/**]
+    branches-ignore:
+      - dependabot/**
   pull_request:
     # The branches below must be a subset of the branches above
-    branches: [develop]
+    branches:
+      - develop
   schedule:
     - cron: '0 21 * * 6'
 
 jobs:
   analyze:
     name: Analyze
     runs-on: ubuntu-latest
-
+    permissions:
+      # required for all workflows
+      security-events: write
     strategy:
       fail-fast: false
       matrix:
         # Override automatic language detection by changing the below list
-        # Supported options are ['csharp', 'cpp', 'go', 'java', 'javascript',
-        # 'python']
-        language: ['python']
+        # Supported options are go, javascript, csharp, python, cpp, and java
```

Looks like we missed alphabetizing this in [skeleton-python-library](https://github.com/cisagov/skeleton-python-library/blob/develop/.github/workflows/codeql-analysis.yml#L33).  Or is there a reason for this particular ordering of options?
```suggestion
        # Supported options are cpp, csharp, go, java, javascript, and python
```

---

The workflow is in large part cribbed from the [example workflow](https://github.com/github/codeql-action#usage) for the Action.

---

So does that mean that you are against alphabetizing this list here and in [skeleton-python-library](https://github.com/cisagov/skeleton-python-library/blob/develop/.github/workflows/codeql-analysis.yml#L33)?

---

I think unless we did heavier editing I'd prefer to leave it as close as possible to the example for now.

---

OK, no problem.